docs/specs/rpa-dg-docassembly-api.json

```diff
@@ -1 +1 @@
ment Assembly API"},"host":"localhost:8080","basePath":"/","tags":[{"name":"form-definition-resource","description":"Form Definition Resource"},{"name":"template-rendition-resource","description":"Template Rendition Resource"}],"paths":{"/api/form-definitions/{templateId}":{"get":{"tags":["form-definition-resource"],"summary":"Retrieves Document Assembly Form Definition (Formly JSON see https://github.com/ngx-formly/ngx-formly).","operationId":"getFormDefinitionUsingGET","produces":["*/*"],"parameters":[{"name":"Authorization","in":"header","description":"Authorization","required":true,"type":"string"},{"name":"ServiceAuthorization","in":"header","required":true,"type":"string"},{"name":"templateId","in":"path","description":"templateId","required":true,"type":"string"}],"responses":{"200":{"description":"OK","schema":{"$ref":"#/definitions/JsonNode"}},"401":{"description":"Unauthorized"},"403":{"description":"Forbidden"},"404":{"description":"Not Found"}},"deprecated":false}},"/api/template-renditions":{"post":{"tags":["template-rendition-resource"],"summary":"Renders a templates using provided values and uploads it to Document Store","operationId":"createTemplateRenditionUsingPOST","consumes":["application/json"],"produces":["*/*"],"parameters":[{"name":"Authorization","in":"header","description":"Authorization","required":true,"type":"string"},{"name":"ServiceAuthorization","in":"header","required":true,"type":"string"},{"in":"body","name":"createTemplateRenditionDto","description":"createTemplateRenditionDto","required":true,"schema":{"$ref":"#/definitions/CreateTemplateRenditionDto"}}],"responses":{"200":{"description":"OK","schema":{"$ref":"#/definitions/CreateTemplateRenditionDto"}},"201":{"description":"Created"},"401":{"description":"Unauthorized"},"403":{"description":"Forbidden"},"404":{"description":"Not Found"}},"deprecated":false}}},"definitions":{"CreateTemplateRenditionDto":{"type":"object","properties":{"formPayload":{"$ref":"#/definitions/JsonNode"},"jwt":{"type":"string"},"outputType":{"type":"string","enum":["PDF","DOC","DOCX"]},"renditionOutputLocation":{"type":"string"},"templateId":{"type":"string"}},"title":"CreateTemplateRenditionDto"},"JsonNode":{"type":"object","properties":{"array":{"type":"boolean"},"bigDecimal":{"type":"boolean"},"bigInteger":{"type":"boolean"},"binary":{"type":"boolean"},"boolean":{"type":"boolean"},"containerNode":{"type":"boolean"},"double":{"type":"boolean"},"empty":{"type":"boolean"},"float":{"type":"boolean"},"floatingPointNumber":{"type":"boolean"},"int":{"type":"boolean"},"integralNumber":{"type":"boolean"},"long":{"type":"boolean"},"missingNode":{"type":"boolean"},"nodeType":{"type":"string","enum":["ARRAY","BINARY","BOOLEAN","MISSING","NULL","NUMBER","OBJECT","POJO","STRING"]},"null":{"type":"boolean"},"number":{"type":"boolean"},"object":{"type":"boolean"},"pojo":{"type":"boolean"},"short":{"type":"boolean"},"textual":{"type":"boolean"},"valueNode":{"type":"boolean"}},"title":"JsonNode"}}}
```

Comment:
isn't this generated automatically? on push to your master branch?

Reply:
I don't know. I will test it today.